docs/Cloud_integrations/Device_Integration_Center/overview.md

```diff
@@ -6,83 +6,153 @@ brokers etc. There are two kind of events:
-variable does not exist e.g. `$notExisting` then it is sent as normal text. Available variables differs from context where
-they are used.
+variable does not exist e.g. `$notExisting` then it is sent as normal text.
+
+=== "Device Lifecycle Variables"
+
+      * `version` *(type: string)* - The version in which events are sent. Any changes will be reflected in it.
+      * `timestamp` *(type: long)* - Time of occurrence of the event in milliseconds.
+      * `endpointName` *(type: string)* - The device endpoint name given at creation.
+      * `domainId` *(type: string)* - The entire ID of the domain in which the device is located.
+      * `domainName` *(type: string)* - The name of the last subdomain without slashes.
+      * `subtype` *(type: string)* - Enumeration type of the event.
+      * `metadata` *(type: json|string)* - Additional details about the event in relation to its subtype.
+
+        !!! Note "Metadata format"
+            For the `JSON` content type, it is an object with possibly nested fields. Whereas for `TEXT` it is a simple string with fields separated by commas.
+
+=== "Device Telemetry Variables"
+
+      * `version` *(type: string)* - The version in which events are sent. Any changes will be reflected in it.
+      * `timestamp` *(type: long)* - Time of occurrence of the event in milliseconds.
+      * `endpointName` *(type: string)* - The device endpoint name given at creation.
+      * `domainId` *(type: string)* - The entire ID of the domain in which the device is located.
+      * `domainName` *(type: string)* - The name of the last subdomain without slashes.
+      * `url` *(type: string)* - LwM2M path (Resource Id) for which changes are occurring.
+      * `value` *(type: string|number)* - The value of the changed LwM2M resource. 
+
+        !!! Note "Value format"
+            Although it can be a number, in the case of `JSON` content type it must be enclosed in quotation marks.
+
+Available variables differs from context where they are used. See [headers](#headers) and [body](#body) for detailed information.
 
 ### Headers
 
-Variables can be used in the `Header value` field (variables defined in `Header key` will not be considered). You can
-enter any header value e.g. some token. If you change type to `$` you will be allowed to easily select some variable.
-However, in `abc` mode, variables can also be used.
+Variables can be used in the `Header value` field. You can
+enter any header value e.g. some token. If you change type to `$` you will be allowed to easily select available variables.
+However, in `abc` mode, variables can also be used with the help of the `$` sign.
+
+!!! Note
+
+    Variables cannot be used in the header key. Thus, any `$` references defined in the `Header key` will not be considered.
 
 ![Custom Headers Dialog](./images/custom-headers-dialog.webp){ width=80% }
 
-For device lifecycle events, there are the following variables:
+=== "Available lifecycle variables"
```

Comment:
Consider marking it as a header, probably...?

Reply:
I would like to keep it as a content tab because:
a) it is more compact
b) more importantly, it clearly shows the difference

![image](https://github.com/AVSystem/developer-zone/assets/89745030/74ac87e1-09d6-4c77-a9e6-037cfaaf33f7)